script/install/debootstrap.bash

```diff
@@ -173,5 +173,47 @@ echo "$chroot_cmd update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-9
 chroot "$ISOLATE_ROOT" update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-9 75
 # gcc 9 done
 
-# let user know that chroot installs are finished
+[ -z "$TRAVIS" ] && bash script/confirm.bash 'Install .Net Core (C#)' && {
+  set -x
+
+  : Installing .Net Core for C#
+  # https://docs.microsoft.com/en-us/dotnet/core/install/linux-package-manager-ubuntu-1604
+
+  : Downloading package info from Microsoft
+  UBUNTU_VERSION=$(lsb_release --release --short)
+  chroot "$ISOLATE_ROOT" wget -q "https://packages.microsoft.com/config/ubuntu/${UBUNTU_VERSION}/packages-microsoft-prod.deb" -O packages-microsoft-prod.deb
+  chroot "$ISOLATE_ROOT" dpkg -i packages-microsoft-prod.deb
+  chroot "$ISOLATE_ROOT" rm packages-microsoft-prod.deb
+  [ "$UBUNTU_VERSION" == "18.04" ] && chroot "$ISOLATE_ROOT" sudo add-apt-repository universe
```

Comment:
According do the [code](https://github.com/dotnet/core/commit/adc8888f3ab2) the 'universe' repository is always required, but only mentioned for Ubuntu 18.04 because in the others it's enabled by default. We might as well enable it unconditionally. (Also, no need for sudo.)